scapy/layers/l2.py

```diff
@@ -67,8 +75,8 @@ def getmacbyip(ip, chainCC=0):
     tmp = [orb(e) for e in inet_aton(ip)]
     if (tmp[0] & 0xf0) == 0xe0:  # mcast @
         return "01:00:5e:%.2x:%.2x:%.2x" % (tmp[1] & 0x7f, tmp[2], tmp[3])
-    iff, a, gw = conf.route.route(ip)
-    if ((iff == scapy.consts.LOOPBACK_INTERFACE) or (ip == conf.route.get_if_bcast(iff))):
+    iff, _, gw = conf.route.route(ip)
+    if ((iff == LOOPBACK_INTERFACE) or (ip == conf.route.get_if_bcast(iff))):
```

Comment:
Please leave static calls to LOOPBACK_INTERFACE, it can get changed by arch/windows, but because we never know in which order the files are going to be loaded, we need to keep static access

Reply:
OK, thanks for this.

Reply:
(fixed)